cmd/okctl/applycluster.go

```diff
@@ -190,6 +214,7 @@ func buildApplyClusterCommand(o *okctl.Okctl) *cobra.Command {
 			context.GithubCredentialsTypeToken,
 		),
 	)
+	flags.BoolVarP(&opts.Quiet, "quiet", "q", false, "reduces output to screen")
```

Comment:
There is something misleading about combining `OKCTL_DEBUG` (verbose) and `--quiet` (silent), or at least to me it isn't obvious why we are doing this.

But we are basically disabling all the fancy timers and spinners, so maybe we could use a flag: `--disable-fancy-output` or something to that effect instead of `--quiet`?